pkg/components/wfbackend/metadata.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wfbackend
+
+import (
+	"time"
+
+	"github.com/microsoft/durabletask-go/backend/sqlite"
+
+	"github.com/dapr/components-contrib/metadata"
+	metadataUtil "github.com/dapr/kit/metadata"
+)
+
+const (
+	defaultFilePath                 = ""
+	defaultOrchestrationLockTimeout = 2 * time.Minute
+	defaultActivityLockTimeout      = 2 * time.Minute
+)
+
+// Metadata represents a set of workflow specific properties.
+type Metadata struct {
+	metadata.Base `json:",inline"`
+}
+
+type sqliteMetadata struct {
```

Comment:
This is specific to a component (SQLite), so should be moved to its own file please

Perhaps it may even make sense to put this business logic into the `cmd/daprd/components/wfbackend_sqlite.go`, so it's all together.